src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -230,7 +222,7 @@ export class ExposureNotificationService {
   private async recordKeySubmission() {
     const currentStatus = this.exposureStatus.get();
     if (currentStatus.type !== 'diagnosed') return;
-    this.exposureStatus.append({needsSubmission: false, submissionLastCompletedAt: getMillisSinceUTCEpoch()});
```

Comment:
I found `getMillisSinceUTCEpoch` is confusing. `.getTime()` already returns UTC. So, there is no need for it.